src/libraries/System.Text.Json/src/Resources/Strings.resx

```diff
@@ -1,17 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
 <root>
-  <!--
-    Microsoft ResX Schema
-
+  <!-- 
+    Microsoft ResX Schema 
+    
```

Comment:
I don't know why the whitespace has changed, I merely changed the string in the latest VS release (17.2.2)

Reply:
I've seen this happening as well. I'd probably just revert the changes to keep the diff lean.

Reply:
Have reverted the whitespace changes and updated the wording with your proposal, but rephrased to remove parentheses on @krwq 's comment.